### ISSUE TITLE
Mismatch "database" field in docs

### ISSUE THREAD

**Issue by FlibustierSparrow** [TimonKK/clickhouse#53]
You have written that field "database" should be in the inner object "config" in docs, but actually It is false.
# In docs
```
const clickhouse = new ClickHouse({
         ....
	config: {
               .....
		database                                : 'my_database_name',
	},
});
``` 
# Must be
```
const clickhouse = new ClickHouse({
         ....
         database                                : 'my_database_name',
	config: {
               .....
	},
});
```

**Comment by TimonKK** [comment]
There is not a bug. You can set database as you wish

**Comment by FlibustierSparrow** [comment]
```
const clickhouse = new ClickHouse({
         ....
	config: {
               .....
		database                                : 'my_database_name',
	},
});
```
This case is not working

**Comment by TimonKK** [comment]
Fix in 2.1.4
